src/status_im/ui/screens/usage_data/styles.cljs

```diff
@@ -41,8 +42,10 @@
    :color          colors/gray})
 
 (def buttons-container
-  {:align-items :center})
+  {:flex-direction :row
+   :align-items :center})
```

Comment:
Can you align values?

Reply:
@jeluard Could you explain a bit more what you want here, please? Not sure about your meaning of alignment. :)

Reply:
Done :)